### PR TITLE
Javascript javascript3 week1

### PR DESCRIPTION
**Pull request by oleksandra-lavryk**
JS3 week1

### CODE REVIEW

File: javascript/javascript3/week1/giphyApp/main.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const wordInput = document.getElementById("search-word");
+const wordAmountInput = document.getElementById("gif-amount");
+const gifContainer = document.getElementById("gif-result-conatiner");
+const apiKey = "dr8wkOEpX1Ws8a5AdjQdA8iBQrZumR3q";
+
+searchBtn.addEventListener("click", () => {
+  if (wordInput.value == "") {
+    alert("Enter a word");
+  } else {
+    (async function () {
```
Comment (Falafelqueen) [inline javascript/javascript3/week1/giphyApp/main.js:11]: You guys didn't even go through async await yet. Amazing !!

File: javascript/javascript3/week1/giphyApp/main.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        const content = await response.json();
+        renderGifs(content.data);
+      }
+    })();
+  }
+});
+
+function renderGifs(gifArr) {
+  gifContainer.innerHTML = "";
+  if (wordAmountInput.value > 0) {
```
Comment (Falafelqueen) [inline javascript/javascript3/week1/giphyApp/main.js:25]: Very cool solution ! 💃

File: javascript/javascript3/week1/weatherApp/main.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  resultBlock.style.opacity = "1";
+  localStorage.setItem("lon", content.coord.lon);
+  localStorage.setItem("lat", content.coord.lat);
+
+  if (showMap.checked) {
+    initMap(content.coord.lon, content.coord.lat);
+  }
+}
+
+function initMap(lon, lat) {
```
Comment (Falafelqueen) [inline javascript/javascript3/week1/weatherApp/main.js:94]: Amazing optional !

**Review by Falafelqueen — APPROVED** [review]
Incredible job Oleksandra, it`s obvious and awesome that you are putting time into your homework and it is paying off ! 💃👑

File: javascript/javascript3/week1/giphyApp/main.js (hunk 1 above)
Comment (oleksandra-lavryk) [reply javascript/javascript3/week1/giphyApp/main.js:11]: @Falafelqueen Thomas gave us this example with async and I tried it in homework 😊
Thanks a lot for your review😊

**Pull request merged**